src/index.html

```diff
@@ -40,15 +38,24 @@
     <script async src="https://www.googletagmanager.com/gtag/js?id=G-MGKFYEV4JG"></script>
     <script>
         window.dataLayer = window.dataLayer || [];
-        function gtag() { dataLayer.push(arguments); }
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
         gtag('js', new Date());
 
         gtag('config', 'G-MGKFYEV4JG');
     </script>
 </head>
 
-<body text="#000000" topmargin="0" leftmargin="0" rightmargin="0" bottommargin="0" marginheight="0" marginwidth="0"
-    onload="document.textform.text.focus();">
+<body text="#000000" topmargin="0" leftmargin="0" rightmargin="0" bottommargin="0" marginheight="0" marginwidth="0" onload="document.textform.text.focus();">
+
+
+    <div id="nav">
+        <div id="logo"><span style="font-weight:bold; font-size:25px;color:#ff3333">Edit</span><span>Pad</span></div>
+    </div>
```

Comment:
@Ashish9836 This nav is not looking good with the current look of the Editpad. We can get rid of this.